app/views/taxons/edit.html.erb

```diff
@@ -10,4 +10,9 @@
       input_html: { class: :select2, multiple: true, include_blank: true } %>
 
   <%= f.submit I18n.t('views.taxons.edit_button'), class: "btn btn-lg btn-success" %>
+
+  <%= link_to 'Delete taxon', taxon_path(taxon.content_id),
+    method: :delete,
+    data: { confirm: I18n.t('messages.views.confirm') },
+    class: 'btn btn-lg btn-danger' %>
```

Comment:
In this specific view, I think Mark mentioned it should be a text link rather than a button. But it's fine in the show page.